docs/website/docs/installation.mdx

```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
 
 ## Set up environment
 
-## Install `dlt` library
\ No newline at end of file
+Make sure you are using **Python 3.8 or 3.9**:
+
+```bash
+python3 --version
+pip3 --version
+```
+
+If not, then please follow the instructions below to install it:
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu">
+
+You can install Python 3.9 with an `apt` command.
+
+```bash
+sudo apt update
+sudo apt install python3.9
```

Comment:
I also noticed that `venv` module is not installed automatically so your instructions to create virtual env will fail. on linux
sudo apt install python3.10-venv

is it the same on brew?